bookings/tests/snapshots/snap_test_api.py

```diff
@@ -6,6 +6,90 @@
 
 snapshots = Snapshot()
 
+snapshots["test_confirm_booking 1"] = [
+    {
+        "agency": {"logo_url": "http://www.agency.com/logo.png", "name": "MaaS Line"},
+        "amount": 12,
+        "currency": "EUR",
+        "customer_type": "Aikuinen",
+        "departures": [
+            {
+                "depart_at": "2021-04-20T01:00:00.000Z",
```

Comment:
Agreed and you guessed it right. Swaggerhub puts automatically milliseconds to fields with `date-time` format and I don't think there's any way to avoid those other than doing it by hand. We don't need really those.

Reply:
I shall remove the milliseconds then. :+1: